OWNERS_ALIASES

```diff
@@ -158,7 +158,6 @@ aliases:
     - cjcullen
     - enj
     - joelsmith
-    - lukehinds
     - micahhausler
     - tabbysable
     - tallclair
```

Comment:
Is it possible for you to update current member list? TY!!

Reply:
Where is the current member list?